api/RTCPeerConnection.json

```
@@ -210,6 +210,349 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "configuration_bundlePolicy_parameter": {
+          "__compat": {
+            "description": "<code>configuration.bundlePolicy</code> parameter",
+            "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcconfiguration-bundlepolicy",
+            "support": {
+              "chrome": {
+                "version_added": "23"
+              },
+              "chrome_android": {
+                "version_added": "57"
+              },
+              "edge": {
+                "version_added": "≤79"
+              },
+              "firefox": {
+                "version_added": "42"
+              },
+              "firefox_android": {
+                "version_added": "42"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": "11"
+              },
+              "safari_ios": {
+                "version_added": "11"
+              },
+              "samsunginternet_android": {
+                "version_added": "7.0"
+              },
+              "webview_android": {
+                "version_added": "≤37"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "configuration_certificates_parameter": {
+          "__compat": {
+            "description": "<code>configuration.certificates</code> parameter",
+            "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcconfiguration-certificates",
+            "support": {
+              "chrome": {
+                "version_added": "23"
```

This was added in https://source.chromium.org/chromium/chromium/src/+/ef4ab6012fddc80992b8848cfc6700e731dc39b1 which maps to Chrome 48, but the feature was enabled in https://source.chromium.org/chromium/chromium/src/+/7318c1bfd0b0d45bc323bc4bcc45c98c4aeebfb6 which is Chrome 49. That also matches the `RTCCertificate` data.

```suggestion
                "version_added": "49"
```